Dockerfile.run

```diff
@@ -1,13 +1,15 @@
+FROM alpine:3.4
 
-FROM    alpine:3.4
-RUN     apk -U add \
-            python \
-            py-pip
 
-COPY    requirements.txt /code/requirements.txt
-RUN     pip install -r /code/requirements.txt
+COPY dist/docker-compose-Linux-x86_64 /usr/local/bin/docker-compose
 
-ADD     dist/docker-compose-release.tar.gz /code/docker-compose
-RUN     pip install --no-deps /code/docker-compose/docker-compose-*
+RUN apk update && apk add --no-cache \
+    curl openssl ca-certificates \
+    && chmod +x /usr/local/bin/docker-compose \
+    && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub \
+    && wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.23-r3/glibc-2.23-r3.apk \
+    && apk add --no-cache glibc-2.23-r3.apk && rm glibc-2.23-r3.apk \
+    && ln -s /lib/libz.so.1 /usr/glibc-compat/lib/ \
+    && ln -s /lib/libc.musl-x86_64.so.1 /usr/glibc-compat/lib
```

Comment:
Minor: please move all the `&&` to the end of the previous lines. That's the style we use in other places.